pkg/operation/botanist/component/etcd/monitoring.go

```diff
@@ -115,6 +115,18 @@ const (
         within the last hour.
       summary: High number of failed etcd proposals
 
+  - alert: KubeEtcd3HighMemoryConsumption
+    expr: sum(container_memory_working_set_bytes{pod="etcd-main-0",container="etcd"}) / sum(vpa_spec_container_resource_policy_allowed{allowed="max",container="etcd", targetName="etcd-main", resource="memory"}) > .5
```

Comment:
Should we make the threshold limit `0.4` (instead of `0.5`) to give the shoot owner more time to reduce the load on ETCD because reducing load on ETCD involves controller/operator optimisation which takes time?

For ETCD, the VPA `maxAllowed` is currently configured to be `30Gi`. So, at `0.5` the threshold would be `15Gi` and at `0.4` it would be `12Gi`. The largest ETCDs in the landscapes have crossed `12Gi` but not `15Gi` yet.

Reply:
I've changed the value to `.4`